packages/components/src/components/tabs/tabs.tsx

```diff
@@ -0,0 +1,46 @@
+import * as TabsPrimitive from '@radix-ui/react-tabs';
+import * as React from 'react';
+
+import { cn } from '~/lib/utils';
+
+const Tabs = TabsPrimitive.Root;
+
+const TabsList = React.forwardRef<
+  React.ElementRef<typeof TabsPrimitive.List>,
```

Comment:
Lets follow convention from other components and import what we need from React instead of using React wildcard.

Reply:
agree and updated